CHANGES.rst

```diff
@@ -1,5 +1,8 @@
 Changelog
 =========
+* next release
```

Comment:
i think we can get away with 1.2, if people do install -U they will get new lib and their stuff will not have to change.
They will just be more accurate, since classes attributes stayed the same from what i see.